content/en/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+In robotics and automation, a _control loop_ is
+a non-terminating loop that regulates the state of a system.
+
+Here is one example of a control loop: a thermostat in a room.
+
+When you set the temperature, that's telling the thermostat
+about your *desired state*. The actual room temperature is the
+*current state*. The thermostat acts to bring the current state
+closer to the desired state, by turning equipment on or off.
+
+In Kubernetes, a _controller_ is a control loop that watches the
+state of your
+{{< glossary_tooltip term_id="cluster" text="cluster">}}, then
+makes or requests changes. Each controller tries to move the current
+cluster state closer to the desired state.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Controller pattern
+
+A controller tracks at least one Kubernetes resource type.
+These [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/)
+and their associated data are the desired state, so the
+controller(s) for that resource are responsible for making the current
+state come closer to that desired state.
+
+The controller might carry the action out itself; more commonly, in Kubernetes,
+a controller will send messages to the
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} that have
+useful side effects.
+
+### Control via API server
+
+The controller for Job is an example of a built-in controller that
+makes all of its changes by interacting with your cluster's API server.
```

Comment:
Could line 45 be reworded, possibly something like:
The Job Controller is an example of a Kubernetes built-in controller. Built-in controllers manage state by interacting with the cluster API server.